src/app/src/util/util.js

```diff
@@ -14,34 +14,33 @@ export function DownloadCSV(data, fileName) {
     }
 }
 
-export const makeUserLoginURL = () => `${process.env.REACT_APP_API_URL}/user-login/`;
-export const makeUserLogoutURL = () => `${process.env.REACT_APP_API_URL}/user-logout/`;
+export const makeUserLoginURL = () => '/user-login/';
+export const makeUserLogoutURL = () => `/user-logout/`;
```

Comment:
This does not need to be a backtick string any longer.

Reply:
I may have fixed this in a commit which did some linting, but I'll double check.

Reply:
Yeah, this has been fixed.